bitnami/redmine/templates/svc.yaml

```diff
@@ -18,30 +19,33 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
-  sessionAffinity: {{ default "None" .Values.service.sessionAffinity }}
+  sessionAffinity: {{ .Values.service.sessionAffinity }}
```

Comment:
If removed `default "None"`, shouldn't this be inside a conditional?

Reply:
Nope. `service.sessionAffinity` is set by default in the values.yaml